labgrid/remote/client.py

```diff
@@ -763,6 +764,11 @@ def power(self):
                         drv = target.get_driver(TasmotaPowerDriver)
                     except NoDriverFoundError:
                         drv = TasmotaPowerDriver(target, name=None)
+                elif isinstance(resource, YKUSHPowerPort):
+                    try:
+                        drv = target.get_driver(YKUSHPowerDriver)
+                    except NoDriverFoundError:
+                        drv = YKUSHPowerDriver(target, name=None)
```

Comment:
This should be a separate commit. Also `YKUSHPowerDriver` needs to be imported first.